packages/predeployed-connector/src/index.ts

```
@@ -0,0 +1,291 @@
+                } as Errors.UNEXPECTED_ERROR;
+
+            case "wallet_addInvokeTransaction":
+                if (!this.account) {
+                    throw {
+                        code: 63,
+                        message: "An unexpected error occurred",
+                        data: "wallet_deploymentData not implemented",
+                    } as Errors.UNEXPECTED_ERROR;
+                }
+
+                let params = call.params as AddInvokeTransactionParameters;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)